### PR TITLE
Add fultons

### PR DESCRIPTION
**Pull request by metalgearsloth**
- Stackable entity that applies fulton to target and after 45 seconds goes to the specified beacon.
- Stackable and somewhat cheap to craft (2 cloth) as it'll be necessary for salvage.

https://github.com/space-wizards/space-station-14/assets/31366439/a69da49a-20bc-43ee-867b-2cfa504dccbf

https://github.com/space-wizards/space-station-14/assets/31366439/4cd38946-7411-41da-bdc1-028f1c9a83da

Video shows:
- Folding
- Invalid fultoning
- Successful fultoning

:cl:
- add: Added fultons for salvagers.

### CODE REVIEW

**Comment by PixelTheKermit** [comment]
Can you fulton crit/sleeping mobs?

**Comment by Tunguso4ka** [comment]
"We're gonna extract him too, boss?"

File: Resources/Prototypes/Research/industrial.yml [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+  name: research-technology-fultons
+  icon:
+    sprite: Objects/Tools/fulton_balloon.rsi
+    state: fulton_balloon
+  discipline: Industrial
+  tier: 1
+  cost: 2500  # Cheap because salvage should really get it.
+  recipeUnlocks:
+    - Fulton
+    - FultonBeacon
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Research/industrial.yml:14]: could probably just make this roundstart in the salvagevend if they need it so bad. If putting fultons in the autolathe is an issue could either make a salvage fab or just make them orderable.

**Comment by Iamgoofball** [comment]
this seems to be missing all the animation work from the /tg/ implementation that actually communicates what's happening to the player

you also shouldn't be able to fulton something without linking to a beacon, and you should be able to pick your beacon from the balloon pack itself

File: Resources/Prototypes/Research/industrial.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Research/industrial.yml:14]: sure

**Pull request merged**